test/Microsoft.Extensions.Configuration.Binder.Test/ConfigurationBinderTests.cs

```diff
@@ -82,7 +82,8 @@ public void GetScalarNullable()
                 {"Boolean", "TRUe"},
                 {"Nested:Integer", "11"}
             };
-            var configurationBuilder = new ConfigurationBuilder(new MemoryConfigurationProvider(dic));
+            var configurationBuilder = new ConfigurationBuilder();
+            configurationBuilder.AddInMemoryCollection(dic);
```

Comment:
Aside from the change (which looks good), creating a configuration from a single provider requires so much ceremony it makes me wish there was a shortcut. But on the other hand it seems not that useful out side of testing.